code/game/objects/buckling.dm

```diff
@@ -22,9 +22,10 @@
 
 /atom/movable/MouseDrop_T(mob/living/M, mob/living/user)
 	. = ..()
-	if(can_buckle && istype(M) && istype(user))
-		if(user_buckle_mob(M, user))
-			return 1
+	if(!iscyborg(src)) //the code for buckling things to cyborgs is inside of robot.dmm, since cyborgs have different conditions for buckling things than most other buckle-able things do
```

Comment:
istype(src) :nauseated_face:

Reply:
@nemvar ?

Reply:
@nemvar the check here is "_**!**_iscyborg(src)", not "iscyborg(src)"
please let this PR go through this is really annoying for a lot of borgs reeeeee

Reply:
Cmon, it's not that much effort to add a proc here and override for borgs.

Reply:
@AnturK Done (I think)